Adding features to pie chart and more functionality to chart header

kalletlak opened the pull request:
1. Qtip react table to pie chart.
2. Transition to/from pie and table view.
3. Improved chart header operations.

kalletlak committed:
Enhancements to pie chart

1. Add react table tip to pie chart
2. Transition between pie and table view

kalletlak committed:
Tuning pie chart, and chart header operations

zhx828 commented on app/scripts/views/components/pieChart/pieChart.js:
```diff
@@ -100,6 +108,18 @@
+          if ( $("#"+v.opts.charDivId).length ) {
+            updatePieLabels();
+          }
+        });
```
I think you also need to add v.chart.on("filtered") ?

kalletlak replied:
This functionality is there at https://github.com/cBioPortal/iViz/pull/40/files#diff-54aac811532d56db419d4d021f8490beL141

zhx828 commented on app/scripts/views/components/pieChart/pieChart.js:
```diff
@@ -121,6 +142,408 @@
+      updateTables();
+    }
+
+    function updateTables() {
```
This function has been changed, please see https://github.com/zhx828/cbioportal/commit/9bb24a309792b0e4ecfeb1204cf2fe177a17fc42

kalletlak committed:
Attach svg images

kalletlak committed:
Fix chart title warning

kalletlak committed:
Enable iViz dashboard for querying single study

kalletlak committed:
Refining chart header

1. Adding more images to chart header
2. Modifying css
3. Tuning header logic a bit

kalletlak committed:
Add missing image
